### ISSUE TITLE
Opal Ring of Nothing?

### ISSUE THREAD

**Issue by Shivsproket** [LightsHope/issues#981]
There seems to be something wrong with the random bonuses that rings roll. Today I found an Opal Ring without a random bonus on it: https://imgur.com/a/H26Dwxe

According to https://web.archive.org/web/20050513174254/http://wow.allakhazam.com:80/item.html?witem=11980 Opal Rings should have a random bonus attached to it.

**Comment by Gyronell** [comment]
have you deleted your WDB folder recently ?

**Comment by Shivsproket** [comment]
Yes I delete it every time I launch the game. I've shown it to other players, this ring doesn't have any stats.

**Comment by Shivsproket** [comment]
Another Opal Ring with a non-blizzlike roll has shown up, this time it's a "of Frozen Wrath": https://prnt.sc/l3gz82

Here is a ring I saw a few weeks back with "of Healing": https://cdn.discordapp.com/attachments/450448285447225345/494689060095262720/unknown.png
